grr/server/grr_response_server/gui/ui/components/client/client.ts

```diff
@@ -49,6 +49,10 @@ export class Client implements OnInit, OnDestroy {
     });
   }
 
+  removeLabel(labelName: string) {
+    this.clientPageFacade.removeClientLabel(labelName);
+  }
```

Comment:
Nit: the argument name in `addLabel` (and pretty much everything else) is `label`, but here it is `labelName`—looks a bit inconsistent.

Reply:
True, will change to `label`